### DISCUSSION TITLE
Segment Shortcuts and Size Estimation

### DISCUSSION THREAD

**Discussion by Jay123210599** [mifi/lossless-cut#2018] [Q&A]
Are there any keyboard shortcuts/hotkeys for any of the segment options?

Also, I want to estimate how large the segments will be before I export them. How do I do that?

**Comment by mifi** [comment]
> Are there any keyboard shortcuts/hotkeys for any of the segment options?

which options? you will find the keyboard shortcuts in the keyboard shortcuts settings dialog



> Also, I want to estimate how large the segments will be before I export them. How do I do that?

you would have to calculate yourself based on their duration and the input file's size/duration, but maybe losslesscut could show an estimate

**Reply by Jay123210599** [reply]
1. For these: 
![Screenshot (677)](https://github.com/mifi/lossless-cut/assets/141977822/4f3cd561-7b8b-4364-98f1-dfad251187a7)
2. How?

**Reply by mifi** [reply]
1 - you should be able to find under Settings -> Keyboard & mouse shortcuts -> Segments and cut points

2 - example:
- your input file is 80.9mb and 32.67s long
- exporting 1 segment from 00:00:00.498 - 00:00:02.827, see duration in right panel 2328ms
- estimated output size: `80.9MB * 2328ms / 32670ms = 5.8 MB`
